<launch>
  
  <!-- Enable baxter robot -->
  <node name="enable_baxter_robot" pkg="baxter_tools" type="enable_robot.py" output="screen" args="-e"/>

  <arg name="fixed_frame" value="camera_depth_frame" />

  <include file="$(find rgbd_launch)/launch/kinect_frames.launch" />

  <node name="skeleton_tracker" pkg="skeleton_markers" type="skeleton_tracker">
    <param name="fixed_frame" value="$(arg fixed_frame)" />
    <param name="load_filepath" value="$(find pi_tracker)/params/SamplesConfigNewOpenNI.xml" />
  </node>
  
<!--
  TODO: Include joint tracker node
-->

  <!-- Show in Rviz -->
  <node name="rviz" pkg="rviz" type="rviz" output="screen" args="-d $(find baxter_teleop)/config/cartesian_controller.rviz"/>
  
</launch>
